gr-blocks/grc/blocks.tree.yml

```diff
@@ -38,6 +38,8 @@
   - blocks_file_meta_source
   - blocks_file_meta_sink
   - blocks_tagged_file_sink
+  - blocks_sigmf_source_minimal
+  - blocks_sigmf_sink_minimal
```

Comment:
It looks like those to files don't get installed.

Reply:
Every blocks*yml in this directory will get installed by cmake, try removing `..../build/gr-blocks/grc` to re-trigger the cmake logic to run.

Reply:
That's great. Is this a new default in all OOTs? I remember adding every block file to the install list in grc/

Reply:
Im not sure if OOTs do this [yet], to they probably should it's two lines of CMake.